setup.py

```diff
@@ -45,7 +45,7 @@
     include_dirs=include_dirs,
     libraries=['nvptxcompiler_static'],
     library_dirs=library_dirs,
-    extra_compile_args=['-Wall', '-Werror'],
+    extra_compile_args=['-Wall', '-Werror', '-std=c++0x'],
```

Comment:
Should we just start with C++ 11 or is there a reason not to do that?

```suggestion
    extra_compile_args=['-Wall', '-Werror', '-std=c++11'],
```

Reply:
Setting C++11 here initially introduced newer symbols into the package and made the resulting wheel incompatible with the lowest manylinux version that the rest of RAPIDS supports on arm. However, I've fixed some unrelated issues with the build process over the course of the day, so let me see if this works now; maybe that was a red herring and I never retested.

Reply:
It was indeed a red herring from one of my earlier iterations, I suspect from when there were issues with the underlying container libs. I've updated to c++11 now.